### PR TITLE
[#187364741] Fix-User should be able to view specific item with review information

### PR DESCRIPTION
**Pull request by Bananayosostene**
getting a single product, only the information about the product is returned.
a buyer need also to get other information such as seller information, reviews, and ratings so that he/she can make an it can help him/her in decision on product

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -52,8 +55,16 @@ app.use(notFoundHandler)
...
 
+Product.associate()
+Review.associate({ Product, User })
+
 const server = http.createServer(app)
 server.listen(PORT)
+
+
+sequelizeConnection.sync({ force: false }).then(() => {
+})
```
Comment (munezerobagira) [inline index.ts]:
```suggestion
```
It is not recommended to use sync

File: index.ts [hunk 2]
```diff
@@ -52,8 +55,16 @@ app.use(notFoundHandler)
 
 app.use(ErrorHandler)
 
+Product.associate()
+Review.associate({ Product, User })
```
Comment (munezerobagira) [inline index.ts:64]:
```suggestion

```
You can use  hasMany or hasOne, belongs to instead, and its better if it done on model instead in index.ts

**Review by munezerobagira — CHANGES_REQUESTED** [review]

File: index.ts (hunk 2 above)
Comment (Bananayosostene) [reply index.ts:64]: Thank it is corrected

File: index.ts (hunk 1 above)
Comment (Bananayosostene) [reply index.ts]: It was removed

**Review by munezerobagira — APPROVED** [review]

**Pull request merged**